css/cssom/CSSGroupingRule-cssRules.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>CSSOM - CSSGroupingRule - cssRules</title>
+    <link rel="help" href="https://drafts.csswg.org/cssom/#the-cssgroupingrule-interface">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script>
+    function create(t) {
+        var style = document.createElement('style');
+        style.appendChild(document.createTextNode('@media all { * {} }'));
+        document.head.appendChild(style);
+        t.add_cleanup(function() {
+            document.head.removeChild(style);
+        });
+
+        assert_true(!!style.sheet, 'setup - sheet defined');
+        assert_equals(
+            style.sheet.cssRules.length, 1, 'setup - grouping rule created'
+        );
+        assert_equals(
+            style.sheet.cssRules[0].cssRules.length, 1, 'setup - rule created'
+        );
+        return style.sheet.cssRules[0];
+    }
+
+    test(function (t) {
+        var groupingRule = create(t);
+        groupingRule.cssRules.wptMarker = 'wpt';
+
+        // `insertRule` is used to prompt non-conforming implementations to
+        // create a new CSSRuleList object. Its behavior is verified by a
+        // dedicated test and should not influence the result of this
+        // particular test.
+        try {
+            groupingRule.insertRule('.foo {}', 0);
+            groupingRule.insertRule('.bar {}', 0);
+            groupingRule.insertRule('.baz {}', 0);
+        } catch (err) {}
+
+        assert_equals(groupingRule.cssRules.wptMarker, 'wpt');
+
+        try {
+            groupingRule.deleteRule('.foo {}', 0);
+            groupingRule.deleteRule('.bar {}', 0);
+            groupingRule.deleteRule('.baz {}', 0);
+        } catch (err) {}
+
+        assert_equals(groupingRule.cssRules.wptMarker, 'wpt');
```

Comment:
This test could pass if the returned object is a new structured clone each time. Add a straight `assert_equals` on the object directly, too.

Reply:
I based this off of [existing CSSOM tests](https://github.com/web-platform-tests/wpt/blob/13b3630dfbea2e842b45a44c523077d0c9d1b08e/css/cssom/CSSStyleSheet.html). Asserting object equality will require creating additional references, so I assumed this wasn't done as a way to promote garbage collection.

Reply:
Ah, yeah, storing a reference would defeat the purpose of testing the effect of GC. But as-is it seems extremely unlikely for a GC to happen in the middle of running this test...

That said, I think this is good enough to merge.